Source/WebKit/UIProcess/ios/WKActionSheetAssistant.mm

```diff
@@ -1026,7 +1026,10 @@ - (void)handleElementActionWithType:(_WKElementActionType)type element:(_WKActiv
         [delegate actionSheetAssistant:self performAction:WebKit::SheetAction::Copy];
         break;
     case _WKElementActionTypeOpen:
-        [delegate actionSheetAssistant:self openElementAtLocation:element._interactionLocation];
+        if (element.URL)
```

Comment:
Is `element.URL` populated outside of the Safari QR code case? If so, we probably want to preserve the simulate click behavior for the other cases.

Reply:
It is;

> we probably want to preserve the simulate click behavior for the other cases.

like we discussed yesterday, there is no actual way to identify the fact it is actually a QR code, so how should I do this?

Reply:
The special case occurs when the URL comes from the `getAlternateURLForImage:` SPI. We could propagate a new flag down to here, to identify that `element.URL` has been populated using the SPI.